packages/presentation/lib/src/analitics/service.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:firebase_analytics/firebase_analytics.dart';
+
+class Analytics {
```

Comment:
Analytics is a service, which consumes events and sends them to the server under the hood. so, there's no difference with any other api service. so, where should we put it?)

And rename it to FirebaseAnalyticsService and create an abstraction for it plz